llm_judge/test_llm_judge.py

```diff
@@ -0,0 +1,263 @@
+              misunderstanding of the topic or question.
+            - Score 2: The answer contains significant inaccuracies, though it shows some understanding of the topic. Key
+              elements of the question are addressed incorrectly.
+            - Score 3: The answer is partially correct but has noticeable inaccuracies or omissions. It addresses the
+              question but lacks depth or precision.
+            - Score 4: The answer is mostly correct, with only minor inaccuracies or omissions. It provides a generally
+              accurate response to the question.
+            - Score 5: The answer is completely correct and thorough. It demonstrates a deep and accurate understanding of
+              the topic, addressing all elements of the question effectively.""",
+        "examples": """
+            Question: What is the capital of France?
+            Score 1: Completely Incorrect
+            Answer: "The capital of France is Berlin."
+            Explanation: This answer is entirely incorrect and irrelevant, as Berlin is the capital of Germany, not France.
+            Score 2: Significantly Inaccurate
+            Answer: "The capital of France is Lyon."
+            Explanation: This answer demonstrates some understanding that the question is about a city in France, but it incorrectly identifies Lyon as the capital instead of Paris.
+            Score 3: Partially Correct
+            Answer: "I think the capital of France is either Paris or Marseille."
+            Explanation: This answer shows partial knowledge but includes a significant inaccuracy by suggesting Marseille might be the capital. Paris is correct, but the inclusion of Marseille indicates a lack of certainty or complete understanding.
+            Score 4: Mostly Correct
+            Answer: "The capital of France is Paris, the largest city in the country."
+            Explanation: This answer is mostly correct and identifies Paris as the capital. The addition of "the largest city in the country" is accurate but not directly relevant to the capital status, introducing a slight deviation from the question's focus.
+            Score 5: Completely Correct and Thorough
+            Answer: "The capital of France is Paris, which is not only the country's largest city but also its cultural and political center, hosting major institutions like the President's residence, the Elysée Palace."
+            Explanation: This answer is completely correct, providing a thorough explanation that adds relevant context about Paris's role as the cultural and political center of France, directly addressing the question with depth and precision.
+                     """,
+    }
+    return prompt_config
+
+
+def test_single_grading_score(prompt_fixture):
+    prompt_config = prompt_fixture
+    q1 = "What is the capital of China?"
+    a1 = "The capital of China is Kongfu"
```

Comment:
i don't understand why this is the answer, you want to test with a wrong answer?
(maybe i am missing something, but the answer is beijing as far as i know)

Reply:
Hi @ZeevRispler , this is testing if the judge can judge the wrong answer.